sdcm/coredump.py

```diff
@@ -53,31 +55,49 @@ def __str__(self):
             return f'CoreDump[{self.pid}, {self.corefile}]'
         return f'CoreDump[{self.pid}]'
 
+    def update(self,
```

Comment:
why do we need this?

Reply:
before this change we created coredumpInfo entity at the end, now we create it at the begging and add data to it once it is possible.